model/oracle_database_disk_group.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022 Sorint.lab S.p.A.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package model
+
+// OracleDatabaseDiskGroup holds the informations about a disk group.
+type OracleDatabaseDiskGroup struct {
+	Hostname      string  `json:"hostname" bson:"hostname"`
```

Comment:
please avoid to use hostname and dbname here, the infos are already in the host object